src/plugins/controls/public/react_controls/control_group/components/control_renderer.tsx

```diff
@@ -135,7 +137,7 @@ export const ControlRenderer = <
 
   return component && isControlGroupInitialized ? (
     // @ts-expect-error
-    <ControlPanel<ApiType> Component={component} uuid={uuid} />
+    <ControlPanel<ApiType> Component={component} uuid={uuid} className={className} />
```

Comment:
Can we not have classname here? You're not actually using it

Reply:
Ah! Good catch - removed [8c45a2a](https://github.com/elastic/kibana/pull/193081/commits/8c45a2a286557497617f6e9781f46de5948395b7)

Reply:
You'll need to remove it from the props, too :)